vulture_os/toolkit/api_parser/sentinel_one_mobile/sentinel_one_mobile.py

```diff
@@ -104,13 +106,13 @@ def execute_query(self, method, url, query=None, sleep_retry=10, timeout=10):
                 continue
             except requests.exceptions.HTTPError:
                 msg = f"Status Code {response.status_code}, waiting {sleep_retry}s before retrying"
-                logger.info(f"[{__parser__}]:execute_query: {msg}", extra={'frontend': str(self.frontend)})
+                logger.error(f"[{__parser__}]:execute_query: {msg}", extra={'frontend': str(self.frontend)})
```

Comment:
this doesn't seem to be an error, if the retries' behaviour is indeed wanted, this is a WARNING at most (in this case, I feel the other exceptions are also concerned)